python/lsst/pipe/supertask/activator.py

```diff
@@ -132,6 +136,46 @@ def activate(self):
         pass
 
 
+def get_inheritance(stask_class, subclasses=None):
+    """
+    Get all inheritance classes for a given task to differentiate
+     between SuperTask, CmdLineTask, etc.
+
+    :param stask_class: the output class from pyclbr containing the sub-class information
+    :return: a string with all sub-classes
+    """
+
+    if subclasses is None:  # First call, subclasses is empty
+        subclasses = []
+    temp_class = stask_class.super[0]  # simple inheritance
+    if isinstance(temp_class, pyclbr.Class):
+        subclasses.append(temp_class.name)
+        subclasses = get_inheritance(temp_class, subclasses)  # look recursively for super classes
+    else:
+        subclasses.append(temp_class)
+    return subclasses
+
+
+class ManualPackage(object):
+    """
+    Mock package class to  perform the task discovery for manually entered path to (Super)Task. It has the
+    needed methods to the task discovery for a manually entered path in the command line
+    """
+
+    def __init__(self, path):
+        """
```

Comment:
`__init__` should not have a docstring. It should be in the class documentation. See http://developer.lsst.io/en/latest/docs/py_docs.html#classes-methods-and-functions